client/i18n/locales/english/intro.json

```
@@ -2064,13 +2064,13 @@
       "lecture-user-interface-design-fundamentals": {
         "title": "User Interface Design Fundamentals",
         "intro": [
-          "In these lecture videos, you will learn about the fundamentals of user interface (UI) design. You will learn about the terms you need to know to communicate with designers, visual heirarchy, scaling, alignment, whitespace, and many more."
+          "In these lecture videos, you will learn about the fundamentals of user interface (UI) design. You will learn about the terms you need to know to communicate with designers, visual heirarchy, scaling, alignment, whitespace, and much more."
         ]
       },
       "lecture-user-centered-design": {
         "title": "User-Centered Design",
         "intro": [
-          "In these lecture videos, you will learn about best practices necessary in designing user-facing features like dark mode, breadcrumbs, modal dialogs, and many more. You will also learn how to coduct user research, user requirements and testing."
+          "In these lecture videos, you will learn about best practices necessary in designing user-facing features like dark mode, breadcrumbs, modal dialogs, and much more. You will also learn how to coduct user research, user requirements and testing."
```

```suggestion
          "In these lecture videos, you will learn about best practices for designing user-facing features like dark mode, breadcrumbs, modal dialogs, and much more. You will also learn how to conduct user research, user requirements and testing."
```
"necessary" implies you can't design without them, but you can. The design will likely be bad, but you can do that.